posthog/models/plugin.py

```diff
@@ -15,6 +15,7 @@ class PluginType(models.TextChoices):
     plugin_type: models.CharField = models.CharField(
         max_length=200, null=True, blank=True, choices=PluginType.choices, default=None
     )
+    is_global: models.BooleanField = models.BooleanField(default=False)  # Whether plugin is installed for all orgs
```

Comment:
Q: Will merging this break plugin server tests again (due to extra cols)?

Reply:
It will, https://github.com/PostHog/plugin-server/pull/230 standing by